CentralComputing/Command.h

```diff
@@ -0,0 +1,62 @@
+#ifndef COMMAND_H
+#define COMMAND_H
+
+#include "Utils.h"
+#include "SafeQueue.hpp"
+#include "Event.h"
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <netdb.h>
+#include <poll.h>
+#include <atomic>
+#include <thread> // NOLINT
+#include <memory>
+#include <mutex>
+#include <sys/ioctl.h>
+
+namespace Command {
+
+  extern SafeQueue<uint16_t> command_queue;
+
+  void put(uint8_t ID, uint8_t Command);
```

Comment:
Looking at where it is used, I think having `put()` accept each parameter individually makes more sense. Otherwise the calling code has to put it into a struct, which is more lines of code.

Reply:
I agree that put makes sense to have the individual pieces instead of putting them into a struct and then calling it.  However, I think get should return the struct as there is already a decent amount of code in place that relies on the stuct after get is called, and although it isn't really a lot of work to change this everywhere it seems unnecessary as the struct doesn't have any overhead over two uint 16s.